template/template/wallet.py

```diff
@@ -0,0 +1,33 @@
+""" Module for wallet utilities """
+from template.definitions import ContractConfig
+
+
+class Wallet:
```

Comment:
Have some fear that giving an actual template we would have to keep it up to date.
I know we shouldn't change this much if at all after this, but still...

Reply:
Considering unittests and mypy helping to detect issues I don't think that keeping the template updated is a big effort and I would like to have an independent (mocked) venue as a reference for the future partners

Reply:
Agree